src/core/ee/vu_interpreter.cpp

```diff
@@ -3,22 +3,38 @@
 #include "vu_interpreter.hpp"
 #include "../errors.hpp"
 
+#define printf(fmt, ...)(0)
+
 void VU_Interpreter::interpret(VectorUnit &vu, uint32_t upper_instr, uint32_t lower_instr)
 {
+    uint8_t upperfd, lowerfs, lowerft;
```

Comment:
Place this swapping logic in its own function that returns a bool (whether or not to swap upper and lower ops)